wp-content/themes/currentorg/inc/business-directory-plugin.php

```diff
@@ -378,6 +378,14 @@ function wpbdp_filter_the_content(){
 	if( wpbdp_check_if_specific_wpbdp_view( array( 'submit_listing', 'login' ) ) ){
```

Comment:
If they want this to only appear on one page or another, we'd want to rearrange this conditional to something like:
```
if submit_listing or login:
    replace post_content
if the specific one:
    add the images
if submid_listing or login:
    add the shortcode